.changeset/moody-plants-count.md

```diff
@@ -0,0 +1,6 @@
+---
+'@shopify/cli-hydrogen': patch
+'@shopify/create-hydrogen': patch
+---
+
+Fix codegen command for js project
```

Comment:
Is a changeset necessary? Won't these packages already be getting released? And there's a line item around codegen for js projects already?

Reply:
Good habit to have? A changeset snippet indicated what changes this commit composed of and what packages it affects. It's kind of odd to rely on some other changeset to make proper version bumps on the right packages. What if that commit with the package version bump I needed got reverted for whatever reason? Then this PR won't get the correct version bumps.

It didn't matter how many changeset snippets contains duplicated version bump of the same package. It's gonna result to a single major, minor or patch bump.

Reply:
I'm just thinking about the DX in the final release notes for each version. I think everything mentioned should be actionable from an external perspective, not like a work log from our team.